src/module/sheet/helpers.ts

```diff
@@ -189,7 +190,13 @@ function createTooltipListener(
             if (options.locked) {
                 game.tooltip.dismissLockedTooltips();
             }
-            game.tooltip.activate(target, { html, ...tooltipOptions });
+            game.tooltip.activate(target, {
+                html,
+                ...tooltipOptions,
+                cssClass: options.themeGroup
+                    ? `${(options.cssClass ?? "").trim()} ${getCurrentTheme(options.themeGroup)}`.trim()
+                    : options.cssClass,
```

Comment:
It's a bit less clear but avoids the IIFE altogether.

Reply:
That's fine, so long as its not a easy vector for possible future bugs